tests/merch.test.ts

```diff
@@ -0,0 +1,90 @@
+import { ForbiddenError } from 'routing-controllers';
+import { UserAccessType } from '../types';
+import { DatabaseConnection, MerchFactory, UserFactory, PortalState } from './data';
+import { ControllerFactory } from './controllers';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('collection editing', () => {
+  test('Only Admin Can Getting Archived Items', async () => {
+    const conn = await DatabaseConnection.get();
```

Comment:
```suggestion

describe('merch collections', () => {
  test('collections can only be archived by admins', async () => {
    const conn = await DatabaseConnection.get();
```

Reply:
For the typing change, am I just making it a Partial type of the MerchItem?

As for the description, I don't think we really test who can archive collections. I changed the test description and can push it with the typing change